### DISCUSSION TITLE
Comparing model output and recorded data

### DISCUSSION THREAD

**Discussion by chenghuzi** [jonescompneurolab/hnn-core#528] [Ideas]
To write tutorials for the new GUI, it seems we need to consider comparing model output and recorded data. Any ideas on how to integrate this into the new GUI? Like the layout, button, logic, etc. Or if it's not of the highest priority, we can leave it aside temporarily. @jasmainak

**Comment by jasmainak** [comment]
we should use this function:

https://jonescompneurolab.github.io/hnn-core/stable/generated/hnn_core.read_dipole.html#hnn_core.read_dipole

to read experimental data for comparison

**Comment by jasmainak** [comment]
You probably need a button in the GUI called "Load dipole"

**Reply by rythorpe** [reply]
From a user's standpoint, it would be nice to make this button distinct from the other buttons related to network simulations.

**Comment by jasmainak** [comment]
By the way, you should store the figure handles in the GUI object. Initialize them to None and then add them to a dictionary after plotting is done:

```py
class HNNGUI:

    def __init__(self):
        self.figs = None

    def _plot_dipole():
       with self.plot_outputs['L']:
           fig = dpl.plot()
           self.figs['L'] = fig
```

something like this.
